sql/src/main/java/io/crate/operation/udf/UserDefinedFunctionMetaData.java

```diff
@@ -63,7 +74,7 @@ public void readFrom(StreamInput in) throws IOException {
         int numArguments = in.readVInt();
         arguments = new ArrayList<>(numArguments);
         for (int i = 0; i < numArguments; i++) {
-            FunctionArgument argument = new FunctionArgument();
+            FunctionArgumentDefinition argument = new FunctionArgumentDefinition();
```

Comment:
i would prefer to add a static method `fromStream` and make the empty constructor private